### ISSUE TITLE
Missing ON DELETE CASCADE

### ISSUE THREAD

**Issue by MarcJose** [ASKBOT/askbot-devel#580]
Hello,  
looks like your code is missing some `ON DELETE CASCADE` which makes it really hard to completely delete a user from the system (spam etc) and the Django-interface seems to be broken as well so that would be the only option and by the way much better coding.

**Comment by evgenyfadeev** [comment]
Yes, I notice the on_delete option is available fro the foreign keys, these could be put to use in the code.

**Comment by martin-bts** [comment]
Starting with Django 2.0 `on_delete` must be set explicitly for foreign key relations.
